tests/spread/general/initialize/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Run snapcraft init
+
+restore: |
+  rm -f ./*.snap
+
+execute: |
+  # 'snapcraft init' should create a usable snapcraft.yaml file
+  snapcraft init
+  snapcraft
```

Comment:
for the sake of completeness, should we check that the `snapcraft.yaml` file has `base: core22` in it?

Reply:
Done!